src/urllib3/util/ssl_.py

```diff
@@ -223,7 +225,14 @@ def create_urllib3_context(
     """
     if SSLContext is None:
         raise TypeError("Can't create an SSLContext object without an ssl module")
-    context = SSLContext(ssl_version or PROTOCOL_TLS)
+
+    # PROTOCOL_TLS is deprecated in Python 3.10 so we pass PROTOCOL_TLS_CLIENT instead.
+    # If it's not available in 'ssl' the value is aliased to PROTOCOL_TLS anyways.
```

Comment:
This comment is no longer true: on Python 3.6+ PROTOCOL_TLS_CLIENT is always available